projects/1.18/assets/justabattery/justabattery/lang/zh_cn.json

```
@@ -1 +1,25 @@
-{}
\ No newline at end of file
+{
+  "itemGroup.justabattery": "只是个电池",
+  "item.justabattery.battery": "电池",
+  "item.justabattery.name.prefix": "%d-单元",
+  "item.justabattery.name.prefix.2": "双节",
+  "item.justabattery.name.prefix.3": "三节",
+  "item.justabattery.name.prefix.4": "四号",
+  "item.justabattery.name.prefix.5": "五号",
+  "item.justabattery.desc.level": "等级：%d",
+  "item.justabattery.desc.tracewidth": "导线宽度：%d（最大放电：%dFE/t）",
+  "item.justabattery.prefix.mode": "模式：",
+  "item.justabattery.name.mode.0": "无",
+  "item.justabattery.desc.mode.0": "在世界中潜行+右击改变",
```

什么叫在世界中

---

就是在游戏中，world 指存档，我认为可省去
```suggestion
  "item.justabattery.desc.mode.0": "潜行+右击改变模式",
```